releases/release-1.22/README.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Kubernetes 1.22 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.22
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.22/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.22/release-team.md)
+* Meeting Minutes: TODO (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.22 Release Calendar][k8s122-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info][Internal Contact Info] (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* Enhancements Tracking Sheet: TODO
+* Bug Triage Tracking Sheet: TODO
+* CI Signal Report: TODO
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.22 milestone](https://github.com/kubernetes/kubernetes/milestone/50)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.22 release cycle is proposed as follows:
+
+- **Monday, April 26th**: Week 1 - Release cycle begins
+- **Tuesday, May 25th**: Week 5 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
```

Comment:
Can we please push this date back? We had previously discussed [doing enhancements freeze earlier](https://github.com/kubernetes/sig-release/issues/1496#issuecomment-807329101) in the release cycle, e.g. in week 2. Week 3 might make for sense for the longer release cycle.

Reply:
Ref: this was discussed with and acked by [chairs and leads](https://docs.google.com/document/d/1Jio9rEtYxlBbntF8mRGmj6Q1JAdzZ9fTDo3ru1HK_LI/edit#heading=h.3zavw6ipqibo) in the March meetings as well as the [PRR team](https://docs.google.com/document/d/10QkXwiZfBL7wBKHHFslooXvGDKHA75jkkgiIXuKFryM/edit#heading=h.nxm6b7vlrocv) in the Mar. 24 meeting.

Reply:
Putting enhancements freeze on week three would put it on the 11th of May right? But the whole of the week before is KubeCon. I'm assuming reviewers are going to be hard to come by during that week.

If so does that mean that realisticly if you want a KEP reviewed for 1.22 you've got next week (week 1, April 26–30th) and the two days before enhancements freeze (week 2, May 10–11th). Is that too tight?

Reply:
Realistically, most SIGs I'm involved in have already started 1.22 planning. In a 4/yr release cadence, enhancements freeze likely would have been next week or the week after. Week 3 doesn't seem that bad to me, particularly if we do it later in the week (is there any reason it has to be M/Tu? Is Thu an option?) I would say May 17/18th at the very latest.

Reply:
@ehashman: what are your thoughts on moving enhancements freeze to May 13th? would it be a burden for PRR reviewers?

Reply:
Not a blocker comment, just want to point out that this will not only affect PRR reviewers but also the Enhancements team, especially the shadows. Ramping up and reviewing ~60+ Keps (an estimated number based on the last release), is going to put a lot of pressure on the team. Just something you should let your shadows know @JamesLaverack :)

Reply:
The enhancements freeze has been updated to May 13th for now.